src/pkg/selectors/selectors.go

```diff
@@ -272,7 +279,7 @@ func selectorAsIface[T any](s Selector) (T, error) {
 		a, err = func() (any, error) { return s.ToSharePointRestore() }()
 		t = a.(T)
 	default:
-		err = errors.New("service not supported: " + s.Service.String())
+		err = clues.Stack(ErrorUnrecognizedService, errors.New(s.Service.String()))
```

Comment:
What is the stack trace for this going to look like? Since we're not wrapping the original error is seems like we could end up with stack traces for both? Stack traces can generally be printed with `%+v` when using `github.com/pkg/errors`

Reply:
Honestly, I haven't quite investigated stack trace outputs.  That's something I can chase down at a later date.  

Sufficed to say: I'll make it useful.

Reply:
for stack traces I'd prefer to have them working sooner rather than later. They've been invaluable for finding issues so losing them would set back debugging efforts quite a bit